src/main/java/edu/umich/its/cpm/MigrationTaskService.java

```diff
@@ -1545,13 +1546,35 @@ public String updateGoogleGroupMembershipFromSite(String siteId, String sessionI
 		String messageId = message.getMessage_id();
 		log.info("begin to upload message " + messageId  + " to Google Group id = " + googleGroupId);
 		
-		// TODO need to call email RFC822 formatter once it is ready
+		// use EmailFormatter to get RFC822 complaint email content
 		String emailContent = message.getJson();
-		status = addEmailToGoogleGroup(googleGroupId, emailContent);
+		HttpServletRequest request = null;
+		AttachmentHandler attachmentHandler = new AttachmentHandler(request);
+		try
```

Comment:
I think you need to use the `attachmentHandler.setEnv(env)`, as the spring autowiring Enviromnet is not work as expected and results in  `env = null`